### ISSUE TITLE
return 1 without STDOUT

### ISSUE THREAD

**Issue by yamatatsu** [Leko/hothouse#91]
## Environment

- Version: 0.4.1
- package manager: yarn
- directory structure: lerna

## Overview
```
[~/Projects/magic-steak|v10.5.0]
‹master› »»»» env DEBUG=hothouse\* hothouse -t secret
  hothouse:cli CLI options are: { _: [],
  help: false,
  version: false,
  'per-package': true,
  p: true,
  perPackage: true,
  bail: true,
  b: true,
  'dry-run': false,
  dryRun: false,
  t: 'secret',
  token: 'secret',
  concurrency: 4,
  ignore: [],
  '$0': '/usr/local/bin/hothouse',
  'package-manager': undefined,
  'repository-structure': undefined } +0ms
  hothouse:Engine dryRun=false +0ms
  hothouse:PackageManagerResolver Detect package manager in: /Users/yamamototatsuya/Projects/magic-steak +0ms
  hothouse:PackageManagerResolver Yarn: matched=true +23ms
  hothouse:RepositoryStructureResolver Detect repository structure in: /Users/yamamototatsuya/Projects/magic-steak +0ms
  hothouse:RepositoryStructureResolver YarnWorkspaces: matched=false +1ms
  hothouse:RepositoryStructureResolver Lerna: matched=true +0ms
[~/Projects/magic-steak|v10.5.0]
‹master› »»»»                                                       1|14:02:04
```

## Actual behavior
return 1 without STDOUT

## Expected behavior
any STDOUT as success or error

**Comment by Leko** [comment]
@yamatatsu Thank you for your reporting.
I found a bug in Engine#run

https://github.com/Leko/hothouse/blob/f84d2fef68d7e46593aa278d970f28a3eb41a944/packages/hothouse/src/Engine.js#L78-L98

Hothouse report nothing when some error occurred in these lines.

**Comment by Leko** [comment]
@yamatatsu I just released a new version of hothouse (`0.4.7`).
And now you can debug what error occurred in your projects.

Please update hothouse to 0.4.7 and tell me the error result.

**Comment by Leko** [comment]
@yamatatsu I closed this issue.
If you still have any trouble, please re-open and describe the problem.
Thanks.
